docs/resources/ui/components/built-in-components/datetime-picker.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: datetime-picker
+title: DateTime Picker
+tags: [Actions, Components]
```

Comment:
Same comment as color picker.

Reply:
Again, this is available as action only.